docs/git-compatibility.md

```diff
@@ -174,15 +174,16 @@ technically possible (though not officially supported) to convert it into a
 co-located repo like so:
 
 ```bash
+# Ignore the .jj directory in Git
+echo '/*' > .jj/.gitignore
 # Move the Git repo
 mv .jj/repo/store/git .git
 # Tell jj where to find it
 echo -n '../../../.git' > .jj/repo/store/git_target
-# Ignore the .jj directory in Git
-echo '/*' > .jj/.gitignore
 # Make the Git repository non-bare and set HEAD
 git config --unset core.bare
-jj new @-
+# Let .git/HEAD point to a valid commit
```

Comment:
It often already does point to a valid commit. I think it's better to say something like

> Convince jj to update .git/HEAD to point to the working-copy commit's parent